### PR TITLE
Moved function that sets browser window size to test support file so …

### PR DESCRIPTION
**Pull request by ldimaggi**
…tests will not have to duplicate the function. Added a new (empty) template file to assist writers in adding tests.

### CODE REVIEW

**Comment by sanbornsen** [comment]
Neat !! @ldimaggi You need to configure your editor for indentation!!

File: src/tests/work-item/work-item-list/test-template.spec.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+describe('Test name here...', function () {
+  var page, items, browserMode;
+
+  beforeEach(function () {
+    testSupport.setBrowserMode('phone');
+    page = new WorkItemListPage();    
+  });
+
+  it('should testsomething here...', function() {
+//    expect(something...).toBe(comething...);
```
Comment (joshuawilson) [inline src/tests/work-item/work-item-list/test-template.spec.js]: is this an example or are you missing a test? a placeholder to be implemented later?

Comment (joshuawilson) [inline src/tests/work-item/work-item-list/test-template.spec.js]: nevermind, I get it. very nice :)

Comment (ldimaggi) [inline src/tests/work-item/work-item-list/test-template.spec.js]: Only a placeholder - thx!

**Comment by joshuawilson** [comment]
@ldimaggi add to whitelist

**Comment by joshuawilson** [comment]
@ldimaggi the last comment is supposed to add you to the list so that the system will auto run the ci server checks.

**Pull request merged**